ambari-server/src/main/resources/common-services/ZEPPELIN/0.7.0/package/scripts/alert_check_zeppelin.py

```diff
@@ -37,7 +37,7 @@
 
 def execute(configurations={}, parameters={}, host_name=None):
   try:
-    pid_file = glob.glob(zeppelin_pid_dir + '/zeppelin-*.pid')[0]
+    pid_file = glob.glob(zeppelin_pid_dir + '/zeppelin-zeppelin-*.pid')[0]
```

Comment:
The second `zeppelin` stands for Zeppelin service's unix user, which can be customised (see below).  So it should not be hard coded here, because that would trigger false alerts.  (The same applies to the 0.6.0 script.)

https://github.com/apache/ambari/blob/622bb9eca3b9dbf91efb6e7d77c8b9bad04e7237/ambari-server/src/main/resources/common-services/ZEPPELIN/0.7.0/package/scripts/master.py#L269-L270